src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,105 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.Scanner;
+
+/**
+ * Encapsulates a storage unit which handles loading Tasks from the
+ * file and saving Tasks in the file.
+ *
+ * @author Sean Chin Jun Kai
+ */
+public class Storage {
+    private static final String DIR_PATH = System.getProperty("user.home") + "/ip/data";
+    private String filePath;
```

Comment:
Good job on setting the variables as private! Also for using final appropriately/

Reply:
Good work on making the file paths private